docs/src/documentation/03-components/07-interaction/collapse/03-accessibility.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/collapse/accessibility/
+---
+
+## Accessibility
+
+The Collapse component has been designed with accessibility in mind, providing collapsible content that is fully keyboard accessible and screen reader compatible.
+
+### Accessibility Props
+
+The following props are available to improve the accessibility of your Collapse component:
+
+| Name                | Type       | Description                                                                                                                                 |
+| :------------------ | :--------- | :------------------------------------------------------------------------------------------------------------------------------------------ |
+| expandButtonLabel   | string     | Specifies the accessible label of the button when the content is collapsed. This should clearly indicate the action to expand the content.  |
+| collapseButtonLabel | string     | Specifies the accessible label of the button when the content is expanded. This should clearly indicate the action to collapse the content. |
+| label               | string     | Defines the visible heading that also serves as the accessible name for the collapsible section.                                            |
+| customLabel         | React.Node | Allows replacing the default heading with custom content while maintaining the click target for keyboard accessibility.                     |
+
+### Automatic Accessibility Features
+
+- The component automatically manages ARIA attributes:
+  - `aria-expanded` is automatically set to `true` or `false` based on the current state.
+  - `aria-controls` is automatically set to reference the collapsible content.
+- Focus management is handled automatically:
+  - Both the label and button are interactive and can receive keyboard focus.
+  - Click and keyboard events are properly managed.
```

Comment:
Maybe this part can be in a separate "Focus Management" section as we already have for other components?

Then you could include there also these sentences that are now part of Keyboard navigation so everything is in one place.

```
- Focus order: Focus moves from the header to the button to the content inside (when expanded).
- When collapsed, focus will skip over any focusable elements inside the collapsed content.
```